app/src/main/java/com/example/therundown/domain/NbaManager.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.therundown.domain
+
+import com.example.therundown.data.Data
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object NbaManager {
+
+    private val retrofit =
+        Retrofit
+            .Builder()
+            .baseUrl("https://free-nba.p.rapidapi.com/")
+            .addConverterFactory(GsonConverterFactory.create())
+            .build()
+
+    private val service = retrofit.create(NbaService::class.java)
```

Comment:
maybe we can do service creatin in one line

Reply:
fixed